src/HttpServer.cc

```diff
@@ -586,7 +586,7 @@ void Http2Handler::start_settings_timer() {
 
 int Http2Handler::fill_wb() {
   if (data_pending_) {
-    auto n = std::min(wb_.wleft(), data_pendinglen_);
+    auto n = (std::min)(wb_.wleft(), data_pendinglen_);
```

Comment:
I'm not sure why VS2015 doesn't work with std::min(a, b), and requires extra parentheses?  It makes many noises in this PR.

Reply:
Maybe, but I prefer add extra parentheses -:)

Reply:
Personally, I don't want to add extra parentheses.  It looks strange to me.